pushy/src/main/java/com/relayrides/pushy/apns/ApnsClient.java

```diff
@@ -493,8 +492,8 @@ public void run() {
 
                                 @Override
                                 public void run() {
-                                    for (final Promise<PushNotificationResponse<T>> responsePromise : ApnsClient.this.responsePromises.values()) {
-                                        responsePromise.tryFailure(new ClientNotConnectedException("Client disconnected unexpectedly."));
+                                    for (final Promise<PushNotificationResponse<ApnsPushNotification>> partialResponsePromise : ApnsClient.this.responsePromises.values()) {
+                                        partialResponsePromise.tryFailure(new ClientNotConnectedException("Client disconnected unexpectedly."));
```

Comment:
I would suggest renaming `partialResponsePromise` back to `responsePromise` here.